src/hooks/useStep.tsx

```diff
@@ -65,12 +65,12 @@ export const useStep = () => {
     }
   };
 
-  const currentStepId = pathname ? parseInt(getLastPartOfPath(pathname)) : FIRST_STEP.id;
+  const currentStepId = pathname ? getLastPartOfPath(pathname) : FIRST_STEP.id;
```

Comment:
Can we use nextjs built-in system instead? In https://github.com/OffchainLabs/arbitrum-orbit-deployment-ui/blob/main/app/deployment/step/%5Bid%5D/page.tsx we should be able to get the current step with
```
const router = useRouter()
const { step }= router.query
```

https://nextjs.org/docs/pages/building-your-application/routing/dynamic-routes

Reply:
My bad, the link was for pages router and not app router. The param is passed directly to the page component:
```
export default function StepPage({ params }: { params: { id: string } }) {
  console.log('P', params.id); // will log the current step (as a string)
```